proposal/shengwen-tw/README.rst

```diff
@@ -0,0 +1,142 @@
+#################################################################
+libqpsolver: a library for solving quadratic programming problems
+#################################################################
+
+Basic information
+#################
+
+* A numerical optimizer for solving quadratic cost functions with or without equality / inequality constraints
+
+* The optimizer will be implemented in C. (Not C++ since author will like to integrate it into embedded devices later.)
+
+* The future goal of this project is to integrate it in to embedded devices, which requires more improvments to
+  speed up the code. I expect myself to finish the first version this semester and able to run it on normal PC.
+
+GitHub: https://github.com/shengwen-tw/libqpsolver
+
+Problem to solve
+################
+
+What is Quadratic Programming?
+------------------------------
+
+.. image:: qp_formula.PNG
+
+We call the variable x **optimization variables** to be minimized,
+
+the first term **objective function** or **cost function**,
+
+the second term **equality constraints**,
+
+and the third term **inequality constraints**.
+
+Why do we need Quadratic Programming?
+-------------------------------------
+
+Quadratic Programming shows up frequently in robotics field like control or trajectory planning.
```

Comment:
Could you give some references for how it is used in control?  Review papers will be helpful.  Books or research papers are good too.

Reply:
Sure, I have few paper in my mind now, I'll add it in the proposal.